src/main/jjtree/net/sf/jsqlparser/parser/JSqlParserCC.jjt

```diff
@@ -647,6 +647,9 @@ Statement SingleStatement() :
             LOOKAHEAD(ShowColumns())
             stm = ShowColumns()
             |
+            LOOKAHEAD(ShowIndex())
```

Comment:
Looking at this now, is the lookahead for each of these production branches really needed?

Reply:
Just did for consistency

Reply:
It slows down the generated Parser and I would be surprised when a normal Lookahead like 2 or 3 was not sufficient (if we need a lookahead at all).

Lookaheads are very expensive!